views/commit.html

```diff
@@ -68,6 +70,11 @@
                             <span class="den">{{ lines.found }}</span>
                         </span>
                     </td>
+                    <td>
+                        <span class="fraction">
+                            <span class="num">{{ lines.hit / lines.found }}</span>
```

Comment:
You'll want to use something like https://github.com/vokal/cvr-view/blob/master/routes/commit.js#L85 and have the value available in the view model. Handlebars doesn't support math operations in the views, so you'd need to use a helper if you wanted to do something similar to this form.